openslides/assignments/static/js/assignments/base.js

```diff
@@ -9,9 +9,13 @@ angular.module('OpenSlidesApp.assignments', [])
     return DS.defineResource({
         name: name,
         useClass: jsDataModel,
+        agendaSupplement: '(Assignment)',
```

Comment:
Proposal: Move the parentheses `(` and `)` out of this file and add it in agenda/base.js. So a translator does not have to translate the parentheses here.